src/smif/cli/__init__.py

```diff
@@ -435,30 +435,38 @@ def execute_model_run(args):
     ----------
     args
     """
-    timestamp = datetime.datetime.fromtimestamp(time.time()).strftime('%Y%m%dT%H%M%S')
-
     LOGGER.info("Loading resolution data")
     load_resolution_sets(args.directory)
 
-    LOGGER.info("Getting model run definition")
-    model_run_config = get_model_run_definition(args.directory, args.modelrun)
+    if args.batchfile:
+        with open(args.modelrun, 'r') as f:
+            model_runs = f.read().splitlines()
+    else:
+        model_runs = [args.modelrun]    
 
-    LOGGER.info("Build model run from configuration data")
-    modelrun = build_model_run(model_run_config)
+    for model_run in model_runs:
+        print("Start model run '" + model_run)
+        timestamp = datetime.datetime.fromtimestamp(time.time()).strftime('%Y%m%dT%H%M%S')
```

Comment:
Should we move timestamp up to be generated once, so it is consistent across model runs in a batch?

Reply:
Done